js/apps/admin-ui/src/realm-settings/EmailTab.tsx

```diff
@@ -113,14 +100,26 @@ export const RealmSettingsEmailTab = ({
     toggleTest();
   };
 
+  const saveForm = async (form: RealmRepresentation) => {
+    try {
+      const savedRealm = { ...realm, ...form };
+      // For default value, back end is expecting null instead of empty string
+      if (savedRealm.smtpServer?.port === "") savedRealm.smtpServer.port = null;
```

Comment:
Instead of handling this here, perhaps we can handle this in the `save` function of the `RealmSettingsTabs` component?